tests/test_utils.py

```diff
@@ -34,17 +32,26 @@ def test_utils_load(self):
         self.assertIsInstance(data[0]["os_version"], str)
         self.assertIsInstance(data[0]["platform"], str)
 
-    # https://github.com/python/cpython/issues/95299
-    @unittest.skipIf(
-        sys.version_info >= (3, 12), "not compatible with Python 3.12 (or higher)"
-    )
-    def test_utils_load_pkg_resource_fallback(self):
-        # By default use_local_file is also True during production
-        # We will not allow the default importlib resources to be used, by triggering an Exception
-        with patch.object(ilr, "files") as mocked_importlib_resources_files:
-            # This exception should trigger the alternative path, trying to use pkg_resource as fallback
-            mocked_importlib_resources_files.side_effect = Exception("Error")
-            data = utils.load()
+    def test_utils_load_from_zipimport(self):
+        d = TemporaryDirectory()
+        filename = Path(d.name, "module.zip")
+        with ZipFile(filename, "w") as z:
+            for file in Path("src").rglob("*"):
+                z.write(file, file.relative_to("src"))
```

Comment:
Maybe this can be astracted to a function that compresses our module and outputs the path to the `.zip`.